tests/conftest.py

```diff
@@ -47,6 +47,14 @@ def _current_function_name():
     return inspect.currentframe().f_back.f_code.co_name
 
 
+@pytest.fixture(scope="function", autouse=True)
+def return_to_original_directory():
```

Comment:
Is this fixture in use anywhere now? 
I guess the same functionality could be obtained with the `monkeypatch.chdir`, which is quite nice 🙂